src/components/BaseGame.jsx

```diff
@@ -23,11 +21,9 @@ class BaseGame extends React.Component {
     contract.web3Contract.playerRollDice(
       rollUnder, { from: accountAddress, value },
       (error, result) => {
-        if (error) {
-          console.error(error);
-        } else {
-          console.log(JSON.stringify(result));
-        }
+        error
+          && console.error(error)
+          || console.log(JSON.stringify(result));
```

Comment:
I guess we better of using a ternary here, what do you think?
```
Compiled with warnings.

./src/components/BaseGame.jsx
  Line 25:  Unexpected mix of '&&' and '||'  no-mixed-operators
  Line 26:  Unexpected mix of '&&' and '||'  no-mixed-operators
```
https://eslint.org/docs/rules/no-mixed-operators